palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/service/AuditService.java

```diff
@@ -15,29 +15,22 @@
  */
 package uk.gov.gchq.palisade.service.palisade.service;
 
-import feign.Response;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import uk.gov.gchq.palisade.service.Service;
 import uk.gov.gchq.palisade.service.palisade.request.AuditRequest;
 import uk.gov.gchq.palisade.service.palisade.web.AuditClient;
 
-import java.net.URI;
-import java.util.concurrent.Executor;
-import java.util.function.Supplier;
-
 public class AuditService implements Service {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(AuditService.class);
-    private final AuditClient client;
-    private final Supplier<URI> uriSupplier;
-    private final Executor executor;
 
-    public AuditService(final AuditClient auditClient, final Supplier<URI> uriSupplier, final Executor executor) {
+    @Autowired
```

Comment:
remove Autowired as audit client has already been passed in

Reply:
done